ee_tests/local_cico_run_EE_tests_ts.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Show command before executing
+set -x
+
+# Stop on error
+set -e
+
+# Export test configuration
+chmod +x config/local_osio.conf.sh
+. ./config/local_osio.conf.sh
+
+mkdir -p dist
+
+# Run and setup Docker image 
+docker run --detach=true --name=fabric8-test --cap-add=SYS_ADMIN \
+          -e OSIO_USERNAME -e OSIO_PASSWORD -e OSIO_URL  \
+          -e OSO_USERNAME -e GITHUB_USERNAME -e TEST_SUITE -e QUICKSTART_NAME -e RELEASE_STRATEGY -e DEBUG -e "API_URL=http://api.openshift.io/api/" \
+          -t -v $(pwd)/dist:/dist:Z fabric-ui-test:latest
+
+# Exec EE tests
+docker exec fabric8-test ./ts-protractor.sh $TEST_SUITE | tee theLog.txt
+
+# Test results to archive
+docker cp fabric8-test:/opt/fabric8-test/target fabric8-test:/opt/fabric8-test/theLog.txt .
+  
+# Shutdown container
+docker rm -f fabric8-test
```

Comment:
`npm image:start` failed for me on `docker cp` which caused that `docker rm -f fabric8-test` was not executed - could we more it/duplicate it?